### ISSUE TITLE
Reaction test command doesn't work on windows

### ISSUE THREAD

**Issue by hrath2015** [reactioncommerce/reaction-cli#33]
`reaction test`  gives the error-->
```
SERVER_TEST_REPORTER is not recognized as an internal or external command,
operable program or batch file.
```
If we can remove the reporter env variable (SERVER_TEST_REPORTER)  based on platform (**in case of windows  let it use default spec reporter**), users can use reaction test.
Right now  to run on windows user need to run command:
 `meteor test --once --full-app --headless --driver-package dispatch:mocha`

Also I tried to run in gitbash on windows but same error.

**Comment by jshimko** [comment]
[Fixed in v0.8.4](https://github.com/reactioncommerce/reaction-cli/commit/edd8ff2d2dca9f8309770bf3f485ebf35995d0cb).  Let me know if that works for you.

**Comment by hrath2015** [comment]
Great it worked thanks... 👍
